Sources/NFCPassportReader/Logging.swift

```diff
@@ -30,3 +28,18 @@ extension Logger {
     static let pace = Logger(subsystem: subsystem, category: "PACE")
 }
 
+// This disables logging, as simplest way to achieve iOS 13 support
```

Comment:
Could you explain a bit more in the comment how is logging related to iOS 13 support?

Reply:
`os.log.Logger` is iOS 14+ API